_includes/head--common.html

```diff
@@ -15,12 +15,17 @@
     {% endif %}
 
     {% if jekyll.environment == "production" %}
-	<!-- OneTrust Cookies Consent Notice start for docs.prebid.org -->
-	<script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js"  type="text/javascript" charset="UTF-8" data-domain-script="031ad285-43ce-45f1-a493-8188aaf238cf" ></script>
-	<script type="text/javascript">
-	function OptanonWrapper() { }
-	</script>
-	<!-- OneTrust Cookies Consent Notice end for docs.prebid.org -->
+      <!-- OneTrust Cookies Consent Notice start for docs.prebid.org -->
+      <script type="text/javascript">
+        // Global Promise we can check to make sure OneTrust is loaded.
+        let resolveOneTrust;
+        window.onetrustLoaded = new Promise((resolve) => resolveOnetrust = resolve);
+        function OptanonWrapper() {
+          resolveOnetrust();
+        }
```

Comment:
@bretg How about instead of wrapping all the onetrust stuff in an interval check, we define a global promise that we can refer to? The real problem is that the `otSDKStub` script loads an additional script `otBannerSdk.js`. Within that additional script is where the `OneTrust.insertHTML` and other functions are defined. The OneTrust documentation is terrible but `OptanonWrapper` is the callback the fires when all OneTrust scripts are loaded. The current interval solution that only checks for `window.OneTrust` isn't sufficient.

Reply:
Beyond my JS skill @fowler446 . If you can get it to work, I'll port it to the banner examples page.